packages/sommelier.rb

```diff
@@ -320,6 +393,8 @@ def self.build
         # stopsommelier
         File.write 'stopsommelier', <<~STOPSOMMELIEREOF
           #!/bin/bash
+          # Quickest way is to use killall on Xwayland
+          pgrep Xwayland &>/dev/null && killall Xwayland
```

Comment:
Instead of `&>/dev/null` use `2>> #{CREW_PREFIX}/var/log/sommelier.log`?

Reply:
Probably should add to `killall Xwayland` too in case killall errors or is missing.

Reply:
> Probably should add to `killall Xwayland` too in case killall errors or is missing.

Isn't that better to leave the way it is? It isn't being backgrounded, so such an error will be visible to the user.

Reply:
Now you are starting to think like me.  I don't mind error output directly to the console.